ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -737,7 +737,7 @@ def test_run_with_stopped_master(self, ipactl):
         """
         returncode, output = run_healthcheck(
             self.master,
-            "ipahealthcheck.meta",
+            "ipahealthcheck.meta.services",
```

Comment:
This line should not be changed upstream, please see https://pagure.io/freeipa/c/61c71e4a62347cd7188e407acc69ce8362857cb4